### PR TITLE
6.1 - Setup project with Travis, Coverage, added README and .gitignore

### PR DESCRIPTION
**Pull request by laetitia-gangloff**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ - export PATH=$HOME/maintainer-quality-tools/travis:$PATH
+ - $HOME/maintainer-quality-tools/travis/travis_install_nightly 6.1
+ - pip install coveralls flake8
+
+services:
+  - postgresql
+
+script:
+    - travis_run_flake8
+    - travis_run_tests 6.1 openerp_test
```
Comment (yvaucher) [inline .travis.yml]: Here you can remove openerp_test it has been integrated in the helper

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+language: python
+python:
+  - "2.7"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+ - git clone https://github.com/gurneyalex/maintainer-quality-tools.git $HOME/maintainer-quality-tools
```
Comment (yvaucher) [inline .travis.yml]: https://github.com/OCA/maintainer-quality-tools.git instead

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ - export PATH=$HOME/maintainer-quality-tools/travis:$PATH
+ - $HOME/maintainer-quality-tools/travis/travis_install_nightly 6.1
```
Comment (yvaucher) [inline .travis.yml]: You can simplify this line with simply:
- travis_install_nightly 6.1

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+ - pip install coveralls flake8
+
+services:
+  - postgresql
```
Comment (yvaucher) [inline .travis.yml]: You can remove those 2 last lines
This is now done by quality tools

**Comment by laetitia-gangloff** [comment]
So rewrite the .travis.yml file ...

Maybe, it will usefull to also update documentation ?

In step 6 of this guide https://github.com/OCA/maintainers-tools/wiki/Migration-Launchpad-%E2%86%92-GitHub for the initialization of travis it points on https://github.com/gurneyalex/maintainer-quality-tools

And I copied the proposal sample.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+[![Build Status](https://travis-ci.org/OCA/product-attribute.svg?branch=6.1)](https://travis-ci.org/OCA/product-attribute)
+[![Coverage Status](https://img.shields.io/coveralls/OCA/product-attribute.svg)](https://coveralls.io/r/OCA/product-attribute?branch=6.1)
```
Comment (yvaucher) [inline README.md]: please use png badge to allow you to specify the version
See https://raw.githubusercontent.com/OCA/connector/7.0/README.md

**Comment by yvaucher** [comment]
@laetitia-gangloff Indeed

> Follow this example for .gitignore, README.md and .coveragerc: https://github.com/OCA/connector

And 

> .travis.yml config for 6.1 and 7.0 branches: https://github.com/gurneyalex/maintainer-quality-tools

are contradictory

**Comment by yvaucher** [comment]
@laetitia-gangloff Thanks for pointing it on the wiki I updated the wiki page

**Comment by laetitia-gangloff** [comment]
@yvaucher 
In https://github.com/OCA/maintainer-quality-tools the Sample travis configuration file should be also updated, no ?

**Comment by yvaucher** [comment]
@laetitia-gangloff Sure but the team isn't correctly set yet on this project so many PR are waiting

And I believe one is already fixing this.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+[![Build Status](https://travis-ci.org/OCA/product-attribute.svg?branch=6.1)](https://travis-ci.org/OCA/product-attribute)
+[![Coverage Status](https://img.shields.io/coveralls/OCA/product-attribute.svg?branch=6.1)](https://coveralls.io/r/OCA/product-attribute?branch=6.1)
```
Comment (bwrsandman) [inline README.md]: Neat! I didn't know there was an svg option on img.shields.io

**Comment by bwrsandman** [comment]
It seems the last test is giving a false green.
The CI is only failing due to pep8

Made a PR to solve this: https://github.com/OCA/maintainer-quality-tools/pull/24

File: README.md (hunk 6 above)
Comment (yvaucher) [inline README.md]: I'm not sure this is working. Isn't option simply ignored?

Comment (bwrsandman) [inline README.md]: Seems like it

**Comment by yvaucher** [comment]
ValueError: No such external ID currently defined in the system: product.menu_main_product

This might be base.menu_product

**Comment by laetitia-gangloff** [comment]
Should I correct all errors, on the different branch ?

**Comment by yvaucher** [comment]
Yes let's merge this one as it settle all config correctly

:+1: 

Tests can be fixed in another PR

**Pull request merged**